tests/src/application/controllers/load-guardian-name.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { type LoadGuardianName } from '@/domain/use-cases'
+import { LoadGuardianNameController } from '@/application/controllers'
+import { success } from '@/application/helpers'
+import { makeFakeServerError, makeLoadGuardianNameUseCase } from '@/tests/utils'
+
+interface SutTypes {
+  sut: LoadGuardianNameController
+  loadGuardianNameStub: LoadGuardianName
+}
+
+const makeSut = (): SutTypes => {
+  const loadGuardianNameStub = makeLoadGuardianNameUseCase()
+  const sut = new LoadGuardianNameController({ loadGuardianName: loadGuardianNameStub })
+  return {
+    sut,
+    loadGuardianNameStub
+  }
+}
+
+describe('LoadGuardianName Controller', () => {
+  it('Should call LoadGuardianName with correct values', async () => {
+    const { sut, loadGuardianNameStub } = makeSut()
+    const getGuardianNameSpy = jest.spyOn(loadGuardianNameStub, 'load')
+    await sut.handle({ userId: 'any_user_id' })
```

Comment:
Maybe you can transform this kind of object, like `{ userId: 'any_user_id' }` in constants to remove duplications of strings.